README.md

```diff
@@ -297,6 +297,24 @@ match change_password
 
 You'll be asked for the new password on all your machines on the next run.
 
+### Manual control
+
+If you for any reason do not have access to the Dev portal, you can still save your certificates and profiles in match.
+
+To save your certificate, run.
+```
+match manual -p path/to/key.p12 -c path/to/my.mobileprovision -r https://github.com/fastlane/certificates -a tools.fastlane.app -y appstore
```

Comment:
Do you think it's possible to implement something like `match import` which just detects all those values from the local keychain and imports everything?